packages/grid/_modules_/grid/models/rows.ts

```diff
@@ -44,9 +44,9 @@ export interface RowModel {
  * @param rowData Row as [[RowData]].
  * @returns A row as [[RowModel]].
  */
-export function createRow(rowData: RowData): RowModel {
+export function createRow(rowData: RowData, index: RowId): RowModel {
```

Comment:
I had a doubt about what row meant, this would have cleared the confusion:

```suggestion
export function createRowModel(rowData: RowData, index: RowId): RowModel {
```